Documentation/docfx.json

```diff
@@ -5,7 +5,7 @@
         {
           "src": "../",
 	        "files": [
-              "**.csproj"
+              "**.sln"
```

Comment:
How is it related to this change PR change?

Reply:
indeed it is not link to this PR